web/handler/admin_group_remove.go

```
@@ -0,0 +1,37 @@
+}
+
+func (h *Handler) removeAdminGroup(w http.ResponseWriter, r *http.Request) {
+	id := RouteInt64Param(r, "groupId")
+	if err := h.storage.RemoveGroup(id); err != nil {
+		serverError(w, err)
+		return
+	}
+	session := request.GetSessionContextKey(r)
+	session.FlashInfo("Group deleted")
+	session.Save(r, w)
+	http.Redirect(w, r, fmt.Sprintf("/admin/groups"), http.StatusFound)
```

golangci-lint (2.12.1)

[error] S1039: unnecessary use of fmt.Sprintf

(staticcheck)